lints/lint_subject_dn_leading_whitespace.go

```diff
@@ -0,0 +1,85 @@
+// lint_subject_dn_leading_whitespace.go
+
+package lints
+
+import (
+	"github.com/zmap/zcrypto/x509"
+	"github.com/zmap/zlint/util"
+	"strings"
+)
+
+type SubjectDNLeadingSpace struct {
+	// Internal data here
+}
+
+func (l *SubjectDNLeadingSpace) Initialize() error {
+	return nil
+}
+
+func (l *SubjectDNLeadingSpace) CheckApplies(c *x509.Certificate) bool {
+	return true
+}
+
+func (l *SubjectDNLeadingSpace) RunTest(c *x509.Certificate) (ResultStruct, error) {
+	for _, country := range c.Subject.Country {
+		if strings.HasPrefix(country, " ") {
+			return ResultStruct{Result: Warn}, nil
+		}
+	}
+	for _, org := range c.Subject.Organization{
+		if strings.HasPrefix(org, " ") {
+			return ResultStruct{Result: Warn}, nil
+		}
+	}
+	for _, unit := range c.Subject.OrganizationalUnit{
```

Comment:
Again. Maybe this is also a sign that something should be structured differently since you're clearly copy and pasting code repeatedly. Inheritance _is_ a thing in golang....